### DISCUSSION TITLE
W3C Solid Community Group and Social Community Group Joint Meeting

### DISCUSSION THREAD

**Discussion by csarven** [solid/specification#594] [Polls]
Draft Agenda: https://hackmd.io/QOpHN7LsRQO5FwLLjD2zTg

**Comment by michielbdejong** [comment]
I don't have a preference so I didn't vote

**Reply by AaronNGray** [reply]
likewise

**Reply by bumblefudge** [reply]
same

**Comment by TallTed** [comment]
Many in the USA will have Friday, Nov 24, as a holiday, as it follows the Federal holiday of Thanksgiving Day, Nov 23 (4th Thursday) and precedes a typical weekend. (This is the day commonly called "Black Friday" for the extra transactions that bring many retail and other sales companies from loss ("red") to profit ("black") for the year.)

These details may not impact the final decision, but it seems worth noting as a contributory factor for absence by some, if 11/24 is chosen.

**Comment by csarven** [comment]
Thanks all. Noted. There were a few more private responses that preferred the week after - either due to conflict on Tuesday or Friday being a holiday to some.

Decided to go with [2023-11-28T16:00:00Z](https://www.w3.org/events/meetings/82f5ae9e-ff66-4a8a-8bd8-42212d627b46/20231128T140000/). Please subscribe to this event for any changes, and propose agenda topics.

**Reply by TallTed** [reply]
[2023-11-28T16:00:00Z](https://www.w3.org/events/meetings/82f5ae9e-ff66-4a8a-8bd8-42212d627b46/20231128T140000/) was not part of the poll, so I did not need to check my conflicts with it.

This new time conflicts with two other sessions on my calendar: [VC WG call](https://www.w3.org/events/meetings/f6342df0-f7b5-4fc9-babd-61e55dc5fc2f/) (where I'll be) and a [FedID CG call](https://www.w3.org/events/meetings/20c345a0-f8cc-4d4e-9e9d-d24f04816a32/).

I will have to live with the session minutes.

**Comment by AaronNGray** [comment]
On Mon, 20 Nov 2023 at 17:45, Ted Thibodeau Jr ***@***.***>
wrote:

> 2023-11-28T16:00:00Z
> <https://www.w3.org/events/meetings/82f5ae9e-ff66-4a8a-8bd8-42212d627b46/20231128T140000/>
> was not part of the poll, so I did not need to check my conflicts with it.
>
> This new time conflicts with two other sessions on my calendar: VC WG call
> <https://www.w3.org/events/meetings/f6342df0-f7b5-4fc9-babd-61e55dc5fc2f/>
> (where I'll be) and a FedID CG call
> <https://www.w3.org/events/meetings/20c345a0-f8cc-4d4e-9e9d-d24f04816a32/>
> .
>

Why these conflicts? Can someone manage this properly?

**Comment by csarven** [comment]
My apologies. I figured since Tuesday still seemed to work for some, I just moved it to the next week. I don't have everyone's calendar so there'll be some conflicts any way. Hope the new poll works.

For anyone that don't have a preference, please pick a date any way.

Let's try to wrap up this poll this week.

**Reply by acoburn** [reply]
All of those dates work for me.

**Reply by TallTed** [reply]
I know, I keep picking nits.

Pick-one radio buttons are generally not recommended for this sort of question.

Pick-as-many-as-all checkboxes are better (not only because you *should* get actual poll responses from people who are good with multiple if not all options).

General-purpose poll tools — even pick-as-many-as-all checkboxes — are overall not so good for scheduling group meetings. [Doodle polls](https://doodle.com/) are built for it and highly recommended for next time.

**Reply by csarven** [reply]
Sure, I don't mind Doodle but figured GitHub would be more of a neutral (common?) space, presumably with less friction to pick an option, and easier to preserve / recall history. I agree re checkboxes but I didn't spot it in Discussions. Could've asked everyone to write it down in a comment too =) Perhaps next time.

In the end, to break a tie, we'll just pick one at random (small print: not so random). No big deal.

**Comment by michielbdejong** [comment]
Still no preference

**Comment by michielbdejong** [comment]
So 5 December?

**Reply by elf-pavlik** [reply]
Yes https://www.w3.org/events/meetings/82f5ae9e-ff66-4a8a-8bd8-42212d627b46/20231128T140000/

**Reply by michielbdejong** [reply]
Thanks for the link! It still says 'tentative' though, so when will it be confirmed, do you know?

**Comment by csarven** [comment]
Resolving poll with 2023-12-05T16:00:00Z. See https://www.w3.org/events/meetings/82f5ae9e-ff66-4a8a-8bd8-42212d627b46/20231128T140000 for call details.

Catch you all there in a few hours.
